infrastructure/models.py

```diff
@@ -323,11 +357,10 @@ class Project(Publishable):
     estimated_project_output_unit = models.PositiveSmallIntegerField(
         blank=True, null=True, choices=ProjectPlantUnits.UNITS
     )
-    project_capacity = models.FloatField(blank=True, null=True, help_text="MW")
+    project_capacity = models.FloatField(blank=True, null=True)
 
-    project_capacity_unit = models.PositiveSmallIntegerField(
-        blank=True, null=True, choices=ProjectPlantUnits.UNITS
-    )
+    project_capacity_unit = models.CharField(blank=True, default='', max_length=20,
+                                             choices=ProjectCapacityUnits.UNITS)
 
     project_capacity_timeframe = models.CharField(blank=True, max_length=10,
```

Comment:
Hmm, looking at the mockup, should this include **annually** as an option? Right now we can only display **3.8 tons per year**

Reply:
The mock-up shows annually but the pick lists we were given do not have that as an option.  I just went with the pick lists.